Exiled.API/Features/Toys/ToysHelper.cs

```diff
@@ -0,0 +1,65 @@
+// -----------------------------------------------------------------------
+// <copyright file="ToysHelper.cs" company="Exiled Team">
+// Copyright (c) Exiled Team. All rights reserved.
+// Licensed under the CC BY-SA 3.0 license.
+// </copyright>
+// -----------------------------------------------------------------------
+
+namespace Exiled.API.Features.Toys
+{
+    using AdminToys;
+    using Mirror;
+
+    /// <summary>
+    /// A helper class for interacting with toys.
+    /// </summary>
+    public class ToysHelper
```

Comment:
Could be renamed into `ToysPrefabs` or something but this name isn't bad.

Reply:
Name is fine.